src/lib/standaloneCssConfigFilename.ts

```diff
@@ -1,8 +1,22 @@
 export function generateStandaloneCssConfigFilename(fileName: string) {
+  return injectFileNamePart(fileName, 'cssConfig');
+}
+
+export function getRelatedStyleParamsFileName(fileName: string) {
+  /**
+   * this is a file name convention with Yoshi Flow Editor
+   * related:
+   * https://github.com/wix-private/yoshi/blob/7fd08397f8fb744a6fcceb38defd5afb90e86ba8/packages/yoshi-flow-editor/src/wrappers/stylesParamsWrapping.ts#L18
+   * https://github.com/wix-private/yoshi/blob/7fd08397f8fb744a6fcceb38defd5afb90e86ba8/packages/yoshi-flow-editor/src/wrappers/stylesParamsWrapping.ts#L27
+   */
+  return injectFileNamePart(fileName, 'stylesParams');
+}
+
+function injectFileNamePart(fileName: string, injectedString: 'cssConfig' | 'stylesParams') {
```

Comment:
i think the name is a little bit confusing no? maybe something like `appendFileName`.

Reply:
renamed